src/components/widgets/spoolman/SpoolSelectionDialog.vue

```diff
@@ -322,10 +330,15 @@ export default class SpoolSelectionDialog extends Mixins(StateMixin, BrowserMixi
     return this.filename || this.$store.state.printer.printer.print_stats.filename
   }
 
-  get currentFile () {
-    const splitFilepath = this.currentFileName.split('/')
-    const filename = splitFilepath.pop()
+  splitFilePath (path: string): [string, string] {
+    const splitFilepath = path.split('/')
+    const filename = splitFilepath.pop()!
     const filepath = splitFilepath.join('/')
+    return [filename, filepath]
+  }
```

Comment:
I believe we can use the internal `getFilePaths` instead of this?

Reply:
good point, should be fixed :)